contracts/VestingEscrowFactory.vy

```diff
@@ -26,23 +26,29 @@ event VestingEscrowCreated:
     token: indexed(address)
     recipient: indexed(address)
     escrow: address
+    escrow_type: uint256 # 0 - simple, 1 - optimised
```

Comment:
Enums are not supported? More compact data type?

Reply:
There are enums https://vyper.readthedocs.io/en/stable/types.html?highlight=enum#enums
But they seem to be overhead here. + This is only input so using uint256 is not a big deal

Reply:
Enum is much more descriptive and easier to read and understand than to read a comment with types' definitions.